MvvmCross/Platforms/Wpf/Presenters/MultiRegionWpfViewPresenter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Windows;
+using System.Windows.Controls;
+using MvvmCross.Platforms.Wpf.Views;
+using MvvmCross.ViewModels;
+using MvvmCross.Views;
+
+namespace MvvmCross.Platforms.Wpf.Presenters
+{
+    public class MultiRegionWpfViewPresenter : MvxWpfViewPresenter
```

Comment:
We don't want multiple presenters anymore. Can you integrate this into the normal presenter?